src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -1154,6 +1154,18 @@ type SynAttribute =
 
         /// The syntax range of the attribute
         Range: range
```

Comment:
It's not written anywhere, but usually this is the last field in a node, and it makes it a bit easier to match it. I'd suggest moving the new fields before this one.

Reply:
Good point.

Reply:
Oh, but these are record fields :).